cmd/gazelle/fix-update.go

```diff
@@ -51,6 +51,7 @@ type updateConfig struct {
 	walkMode       walk.Mode
 	patchPath      string
 	patchBuffer    bytes.Buffer
+	diffHeader     string
```

Comment:
Please let me know if you are OK with this variable name, or if you would like me to rename it to be `diffHeaderStyle` to be more clear.